source/license.md

```diff
@@ -0,0 +1,21 @@
+---
+title: License
+---
+# License
```

Comment:
Extra # here

Reply:
Also on beta.openssl.org the title is duplicated.

Reply:
Ah, it was a bad idea to use `title:`, as this is understood by pandoc.  I've just submitted #350, which uses `breadcrumb` instead, which is actually what it's used for...  I'll have to adjust all the pages now...

Reply:
I hope I've fixed it, together with #350.

Reply:
There is still the extra `#`

Reply:
Or you need to drop the `--------` under `# License`

Reply:
Done